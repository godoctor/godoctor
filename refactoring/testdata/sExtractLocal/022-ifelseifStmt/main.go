package main

import "fmt"

func main() {
	x := 5
	if x < 0 {

	} else if x < 10 { // <<<<< extractLocal,9,11,9,17,newVar,pass
		fmt.Println("divisible by 5:")
	}
}
