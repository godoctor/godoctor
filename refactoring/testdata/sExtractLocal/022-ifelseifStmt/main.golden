package main

import "fmt"

func main() {
	x := 5
	newVar := x < 10
	if x < 0 {

	} else if newVar { // <<<<< extractLocal,9,11,9,17,newVar,pass
		fmt.Println("divisible by 5:")
	}
}
